### DISCUSSION TITLE
ticket printer compatibility

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3667] [General]
Hello Pretix team.

Is there any standard ticket printer (like laser, not thermal) that I can use with Pretix ?
Or perhaps a thermal printer which is cheaper than Boca Lemur ?

Thank you in advance

**Comment by raphaelm** [comment]
Are you talking about just pretix itself here? If you print PDF files on your computer you can work with any printer your computer supports.

Only if you want to print through our pretixPRINT/pretixSCAN/pretixPOS apps, it needs to be a printer supported by us.

**Reply by alemao8** [reply]
Sorry for not clarifying this.
I mean through pretixPRINT/pretixSCAN/pretixPOS apps.

**Reply by raphaelm** [reply]
Our recommended printers are only the ones listed on https://pretix.eu/about/en/hardware/pos and https://pretix.eu/about/en/hardware/badge

Theoretically, any of the printer types listed here can work:
https://github.com/pretix/pretixprint-android#printer-compatibility

But we might not be able to provide support for it.

**Reply by alemao8** [reply]
ok I see. Thank you

**Reply by alemao8** [reply]
Hello.

We are more closely on Zebra label printers from this list https://github.com/pretix/pretixprint-android#printer-compatibility
Is there any printer command language compatibility I will have to take into consideration ?
I would like to plug a printer that I won't have to make adjustments in the code/drivers in order to work ?

Thank you in advance
